src/data/fintype/basic.lean

```diff
@@ -1390,6 +1390,59 @@ lemma finset.univ_map_embedding {α : Type*} [fintype α] (e : α ↪ α) :
   univ.map e = univ :=
 by rw [←e.equiv_of_fintype_self_embedding_to_embedding, univ_map_equiv_to_embedding]
 
+/-- There exists a bijection between two types of the same finite cardinality, which extends a given
+bijection on a given finset. -/
+lemma finset.exists_equiv_extend_of_card_eq [fintype α] [decidable_eq β] {t : finset β}
+  (hαt : fintype.card α = t.card) {s : finset α} :
+  ∀ {f : α → β} (hfst : s.image f ⊆ t) (hfs : set.inj_on f s),
+    ∃ g : α ≃ t, ∀ i ∈ s, (g i : β) = f i :=
+begin
+  classical,
+  apply @finset.induction α
```

Comment:
Can you avoid the explicitness here by using `induction s using finset.induction_on`? `apply` is usually a bad choice for eliminators compared to even `refine`. ` induction ` will let you drop the intros too

Reply:
Can you please transfer this comment to the prereq PR #16479?